Pages/Shared/_Layout.cshtml

```diff
@@ -44,13 +46,13 @@
 
     <footer class="border-top footer text-muted">
         <div class="container">
-            &copy; 2024 - XML_Project - <a asp-area="" asp-page="/Privacy">Privacy</a>
+            &copy; @DateTime.Now.Year - XmlProject - <a asp-area="" asp-page="/Privacy">Privacy</a>
         </div>
     </footer>
 
-    <script src="~/lib/jquery/dist/jquery.min.js"></script>
-    <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-    <script src="~/js/site.js" asp-append-version="true"></script>
+    <script src="~/lib/jquery/dist/jquery.min.js" defer></script>
```

Comment:
Add defer or async attributes to script tags for better loading performance:
Optimizing script loading enhances user experience.
Optimized script loading for better performance.